src/ui_parts/handles_manager.gd

```diff
@@ -636,21 +643,20 @@ var was_handle_moved := false
 var should_deselect_all = false
 
 func _unhandled_input(event: InputEvent) -> void:
+	if not visible:
+		return
+	
 	var snap_enabled := GlobalSettings.save_data.snap > 0.0
 	var snap_size := absf(GlobalSettings.save_data.snap)
 	var snap_vector := Vector2(snap_size, snap_size)
-	if not visible:
-		return
 	
 	if event is InputEventMouseMotion:
 		should_deselect_all = false
 		var event_pos: Vector2 = event.position / Indications.zoom +\
 				get_node(^"../..").view.position
 		if dragged_handle != null:
 			# Move the handle that's being dragged.
-			var new_pos := SVG.root_tag.world_to_canvas(event_pos)
-			if snap_enabled:
-				new_pos = new_pos.snapped(snap_vector)
+			var new_pos := dragged_handle.transform.affine_inverse() * (event_pos if !snap_enabled else event_pos.snapped(snap_vector))
```

Comment:
Moving handles doesn't work with viewbox anymore.
I tried putting the new position calculation inside the old `SVG.root_tag.world_to_canvas( ... )`, which accounts for all except the O1 O2 values for some reason.

Reply:
Sorry, I'm not quite sure what you mean. I just tried using viewbox and it seems fine, but I'm not too familiar with viewbox so I might just be missing it. What's happening that's unexpected?

Reply:
Try it with a decoupled viewbox, i.e. different width and height than the viewbox.

Reply:
https://github.com/MewPurPur/GodSVG/assets/142301391/09f18d65-ed39-41fe-93e2-7a0b3ea4db6a